test/parallel/test-crypto-hash-stream-pipe.js

```diff
@@ -30,11 +30,17 @@ const crypto = require('crypto');
 
 const stream = require('stream');
 const s = new stream.PassThrough();
-const h = crypto.createHash('sha1');
-const expect = '15987e60950cf22655b9323bc1e281f9c4aff47e';
+const h = crypto.createHash('sha3-512');
```

Comment:
This fails when linked against openssl 1.1.0: https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu1804_sharedlibs_openssl110_x64/24809/

```
15:30:02 not ok 393 parallel/test-crypto-hash-stream-pipe
15:30:02   ---
15:30:02   duration_ms: 0.138
15:30:02   severity: fail
15:30:02   exitcode: 1
15:30:02   stack: |-
15:30:02     internal/crypto/hash.js:33
15:30:02       this._handle = new _Hash(algorithm);
15:30:02                      ^
15:30:02     
15:30:02     Error: Digest method not supported
15:30:02         at new Hash (internal/crypto/hash.js:33:18)
15:30:02         at Object.createHash (crypto.js:101:10)
15:30:02         at Object.<anonymous> (/home/iojs/build/workspace/node-test-commit-linux-containered/test/parallel/test-crypto-hash-stream-pipe.js:33:18)
15:30:02         at Module._compile (internal/modules/cjs/loader.js:778:30)
15:30:02         at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
15:30:02         at Module.load (internal/modules/cjs/loader.js:653:32)
15:30:02         at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
15:30:02         at Function.Module._load (internal/modules/cjs/loader.js:585:3)
15:30:02         at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
15:30:02         at startup (internal/bootstrap/node.js:283:19)
15:30:02   ...
```

Reply:
oops, I'll amend that shortly. Thanks!

Reply:
sorry that shortly turned into 13 days :[. Should be working now. Thanks!